x-pack/legacy/plugins/infra/public/hooks/use_saved_view.ts

```diff
@@ -61,8 +80,9 @@ export const useSavedView = <ViewState>(defaultViewState: ViewState, viewType: s
     saveView,
     loading,
     deletedId,
-    errorOnFind,
-    errorOnCreate,
+    createdId,
+    findError: errorOnFind,
+    createError,
```

Comment:
I thought we changed these from `createError` and `findError` in the saved view PR because they sounded like they were functions. I think I still prefer the `errorOn<TYPE>` names.